modules/apps/27-interchain-accounts/controller/keeper/migrations_test.go

```diff
@@ -29,11 +31,10 @@ func (suite *KeeperTestSuite) TestMigratorMigrateParams() {
 			func() {
 				suite.chainA.GetSimApp().ICAControllerKeeper = icacontrollerkeeper.NewKeeper(
 					suite.chainA.Codec,
-					runtime.NewKVStoreService(suite.chainA.GetSimApp().GetKey(icacontrollertypes.StoreKey)),
+					runtime.NewEnvironment(runtime.NewKVStoreService(suite.chainA.GetSimApp().GetKey(icacontrollertypes.StoreKey)), log.NewNopLogger()),
```

Comment:
I guess we'll have to do some upgrade doc changes for this. Are we tracking that anywhere?

Reply:
Typically in an epic we'd have for an sdk upgrade we'd have a bullet point for keeping track of this.